sdk/python/kfp/cli/pipeline.py

```diff
@@ -66,19 +67,24 @@ def upload(ctx, pipeline_name, package_file):
 def upload_version(ctx, package_file, pipeline_version, pipeline_id=None, pipeline_name=None):
     """Upload a version of the KFP pipeline"""
     client = ctx.obj["client"]
+    output_format = ctx.obj["output"]
     if bool(pipeline_id) == bool(pipeline_name):
         raise ValueError("Need to suppy 'pipeline-name' or 'pipeline-id'")
     if pipeline_name!=None: 
         pipeline_id = client.get_pipeline_id(name=pipeline_name)
         if pipeline_id==None: 
             raise ValueError("Can't find a pipeline with name: %s" % pipeline_name)
-    logging.info("The pipeline id is: %s" % pipeline_id)
+    if output_format == OutputFormat.table.name:
+        # For other output formats users should be able to pipe the output to other programs.
```

Comment:
Perhaps we should mention this piece of logic will happen `_display_pipeline_version` for other formats? Feel free to correct me if I misunderstood this line.

Reply:
I would suggest removing the log message considering the following display (printing) of the pipeline details is a clear message for users that the command run is successful.

Reply:
Yes sounds good to me. My point is just to make this behavior explicit in this comment line.

Reply:
I will remove the log messages and force push the commit now.

Reply:
@numerology Done. Thanks for the quick reply :-) Good work on the project. Big fan.